### PR TITLE
[Update] How to Deploy the Elastic Stack on Kubernetes

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- Updated the instructions to create a LKE cluster
- Updated the screenshots the UI changed for Kibana
- Rephrased to avoid your, will, once

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 80f1d9cb53d23ea300dd08d717e5bb9c44199945

https://deploy-preview-3449--nostalgic-ptolemy-b01ab8.netlify.app

File: docs/kubernetes/how-to-deploy-the-elastic-stack-on-kubernetes/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 author:
-  name: Tyler Langlois
-  email: ty@tjll.net
+  name: Linode
```
Comment (andystevensname) [inline docs/kubernetes/how-to-deploy-the-elastic-stack-on-kubernetes/index.md:3]: Though we are updating the article, we should keep the author name and email.

File: docs/kubernetes/how-to-deploy-the-elastic-stack-on-kubernetes/index.md [hunk 2]
```diff
@@ -209,45 +199,45 @@ TEST SUITE: None
...
 
-1.  A page with the index pattern details will then be shown. Click the **Discover** compass icon from the sidebar to view incoming logs.
+1.  A page with the index pattern details appears. Open the menu, then go to **Kibana > Discover** to view incoming logs.
 
-    ![Kibana Select Discover](kibana-to-discover-tab.png "Kibana Select Discover")
+    ![Kibana Select Discover](kibana-discover.png "Kibana Select Discover")
 
-1.  The Discover page provides a realtime view of logs as they are ingested by Elasticsearch from your Kubernetes cluster. The histogram provides a view of log volume over time, which by default, spans the last 15 minutes. The sidebar on the left side of the user interface displays various fields parsed from json fields sent by Filebeat to Elasticsearch.
+1.  The Discover page provides a realtime view of logs as they are ingested by Elasticsearch from the Kubernetes cluster. The histogram provides a view of log volume over time, which by default, spans the last 15 minutes. The sidebar on the left side of the user interface displays various fields parsed from json fields sent by Filebeat to Elasticsearch.
```
Comment (andystevensname) [inline docs/kubernetes/how-to-deploy-the-elastic-stack-on-kubernetes/index.md:232]: json should be JSON

File: docs/kubernetes/how-to-deploy-the-elastic-stack-on-kubernetes/index.md [hunk 3]
```diff
@@ -257,76 +247,7 @@ When searching in the filters box, field names and values are auto-populated.
...
-    ![Kibana Refresh Fields Confirm](kibana-confirm-refresh.png "Kibana Refresh Fields Confirm")
-
-1.  Navigate to the "Discover" page.
-
-    ![Kibana Select Discover](kibana-to-discover-tab.png "Kibana Select Discover")
-
-1.  Filter for `kibana` containers again, scroll down, and expand a log document. Note that various fields have been parsed into the `kibana` field, such as `kibana.req.method`, indicating which HTTP verb was issued for a request for Kibana.
-
-    ![Kibana Parsed Fields](kibana-kibana-fields.png "Kibana Parsed Fields")
+1.  Look closely at the `message` field in the log representation and note that the text field is formatted as json. While the terms in this field can be searched with free text search terms in Kibana, parsing the field generally yields better results.
```
Comment (andystevensname) [inline docs/kubernetes/how-to-deploy-the-elastic-stack-on-kubernetes/index.md:250]: json should be JSON

**Pull request merged**